server-adapters/resteasy-reactor-netty/src/main/java/org/jboss/resteasy/plugins/server/reactor/netty/ReactorNettyJaxrsServer.java

```diff
@@ -92,33 +88,35 @@ public ReactorNettyJaxrsServer start() {
       log.info("Starting RestEasy Reactor-based server!");
       serverHelper.checkDeployment(deployment);
 
+      //EventLoopGroup eventLoopGroup = new NioEventLoopGroup(1);
       String appPath = serverHelper.checkAppDeployment(deployment);
       if (appPath != null && (root == null || "".equals(root))) {
          setRootResourcePath(appPath);
       }
 
-      final Handler handler = new Handler();
-
-      HttpServer svrBuilder = HttpServer
-              .create()
+      HttpServer svrBuilder =
+          HttpServer.create()
               .tcpConfiguration(this::configure)
-              .port(configuredPort)
-              .handle(handler::handle);
+              .port(configuredPort);
 
-      if (sslContext != null) {
-         svrBuilder = svrBuilder.secure(sslContextSpec -> sslContextSpec.sslContext(toNettySSLContext(sslContext)));
+      if( sslContext != null ) {
+         svrBuilder = svrBuilder.secure(
+                 sslContextSpec -> sslContextSpec.sslContext(toNettySSLContext(sslContext))
+         );
       }
+
       if (hostname != null && !hostname.trim().isEmpty()) {
          svrBuilder = svrBuilder.host(hostname);
       }
 
-      if (Boolean.parseBoolean(System.getProperty("resteasy.server.reactor-netty.warmup", "true"))) {
```

Comment:
@abhijith-prabhakar I think some of these changes are unintentional.  Will you have time to sort these out?